_tools/addonVersion_schema.json

```
@@ -0,0 +1,236 @@
+            "default": "",
+            "description": "The English description of the addon that will be displayed for the addon.",
+            "examples": [
+                "Makes doing XYZ easier"
+            ],
+            "title": "The description (en) of the addon",
+            "type": "string"
+        },
+        "homepage": {
+            "$id": "#/properties/homepage",
+            "default": "",
+            "description": "If the addon has a homepage where users can get more information about the addon, you can specify it here.",
+            "examples": [
+                "https://github.com/nvaccess/addon-store-submission"
+            ],
+            "title": "The homepage URL for the addon.",
+            "type": "string"
+        },
+        "minNVDAVersion": {
+            "$id": "#/properties/minNVDAVersion",
+            "default": {},
+            "description": "The addon will not work with versions of NVDA prior to this version.",
+            "examples": [
+                {
+                    "major": 2019,
+                    "minor": 3,
+                    "patch": 0
+                }
+            ],
+            "required": [
+                "major",
+                "minor",
+                "patch"
+            ],
+            "title": "The minNVDAVersion required for the addon",
+            "type": "object",
+            "properties": {
+                "major": {
+                    "$id": "#/properties/minNVDAVersion/properties/major",
+                    "default": 0,
+                    "description": "'major' in major.minor.patch",
+                    "examples": [
+                        2019
+                    ],
+                    "title": "The 'major' part of the version number.",
+                    "type": "integer"
+                },
+                "minor": {
+                    "$id": "#/properties/minNVDAVersion/properties/minor",
+                    "default": 0,
+                    "description": "The 'minor' part in major.minor.patch",
+                    "examples": [
+                        3
+                    ],
+                    "title": "The 'minor' part of the version number.",
+                    "type": "integer"
+                },
+                "patch": {
+                    "$id": "#/properties/minNVDAVersion/properties/patch",
+                    "default": 0,
+                    "description": "The 'patch' part in major.minor.patch",
+                    "examples": [
+                        0
+                    ],
+                    "title": "The 'patch' part of the version number.",
+                    "type": "integer"
+                }
+            }
+        },
+        "lastTestedVersion": {
+            "$id": "#/properties/lastTestedVersion",
+            "type": "object",
+            "title": "The lastTestedVersion schema",
+            "description": "An explanation about the purpose of this instance.",

```

```suggestion
            "description": "The add-on has been tested up to and including this version of NVDA",
```

Or something like this?